printer_mods/D4rkCelerity/Raspberry_Pi_Zero_Bracket/README.md

```diff
@@ -2,5 +2,12 @@ Raspberry Pi Zero Bracket
 
 This is a modified version of the raspberrypi_bracket for the VORON Trident. Use this bracket along with a pcb_din_clip_v2 to mount a Raspberry Pi Zero to a DIN rail.
 
-Use the same type and quantity of fasteners from the BOM to mount the bracket to din clip and then the RPi Zero to the bracket.
+DIN Rail Install:
+
+ The STL file for the pcb_din_clip_v2 is located on GitHub at VoronDesign/Voron-Trident/STLs/ElectronicsBay/
```

Comment:
Can you make this a hyperlink so that people can click on it and go straight to the necessary part?